todos/003-complete-p1-prd-schema-versioning.md

```
@@ -0,0 +1,94 @@
+### v0.2.5 — Foundational Pieces (NEW)
+- [ ] **PRD Schema Versioning** — Prevent migration issues
+```
+
+**Issue:**
+- v0.1 already creates PRD files (without versioning)
+- Adding versioning later creates backward compatibility debt
+- Migration cost increases with number of existing PRDs
+
+**Recommended placement:**
+```markdown
+### v0.1.5 — CLI Foundation
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)